README.md

```diff
@@ -144,81 +144,146 @@ The old repo is no longer the recommended approach for running and extending the
+| `MECH_MARKETPLACE_ADDRESS` | `str`  | `"0x735FAAb1c4Ec41128c367AFb5c3baC73509f70bB"`                                                                                                                                                                                                                      | Marketplace for posting and delivering requests served by agent mechs. |
+| `HASH_CHECKPOINT_ADDRESS`           | `str` | `0x694e62BDF7Ff510A4EE66662cf4866A961a31653`                                                                                                                                                         | Address of a contract recording metadata.                                         |
 | `AGENT_REGISTRY_ADDRESS`   | `str`  | `"0xE49CB081e8d96920C38aA7AB90cb0294ab4Bc8EA"`                                                                                                                                                                                                                      | Smart contract which registers the agents.                             |
-| `MECH_MARKETPLACE_ADDRESS` | `str`  | `"0x4554fE75c1f5576c1d7F765B2A036c199Adae329"`                                                                                                                                                                                                                      | Marketplace for posting and delivering requests served by agent mechs. |
-| `MECH_TO_SUBSCRIPTION`     | `dict` | `{"0x77af31De935740567Cf4fF1986D04B2c964A786a":{"tokenAddress":"0x0000000000000000000000000000000000000000","tokenId":"1"}}`                                                                                                                                        | Tracks mech's subscription details.                                    |
-| `MECH_TO_CONFIG`           | `dict` | `{"0xFf82123dFB52ab75C417195c5fDB87630145ae81":{"use_dynamic_pricing":false,"is_marketplace_mech":false}}`                                                                                                                                                          | Tracks mech's config.                                                  |
- 
-The rest of the common environment variables are present in the [service.yaml](https://github.com/valory-xyz/mech/blob/main/packages/valory/services/mech/service.yaml), which are customizable too.
+| `LOG_DIR`           | `str` |                                                                                                                                                          | Path for storing the logs.                                         |
 
+⚠️ The variables `CONNECTION_LEDGER_CONFIG_LEDGER_APIS_GNOSIS_ADDRESS` and `CONNECTION_LEDGER_CONFIG_LEDGER_APIS_ETHEREUM_ADDRESS` are expected to be identical. The part `API_KEY` can be changed with your own API KEY. You can also use any other RPC.
```

Comment:
```suggestion
:warning: The variables `CONNECTION_LEDGER_CONFIG_LEDGER_APIS_GNOSIS_ADDRESS` and `CONNECTION_LEDGER_CONFIG_LEDGER_APIS_ETHEREUM_ADDRESS` are expected to be identical. The part `API_KEY` can be changed with your own API KEY. You can also use any other RPC.
```

Reply:
Same comment as before. The configuration should be updated so that the same name is used for both. It does not make sense to ask the user to fill in the same value twice.